aytests/users_no_uid.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e -x
+# Checking if no error has been reported for not defined user UIDs. (bnc#996823)
+DOUBLE_UID=1
+zgrep "Found users in profile with equal <uid>."\
```

Comment:
Why do you need a `DOUBLE_UID` var (maybe I'm missing something)?

```shell
zgrep "Found users in profile with equal <uid>." /var/log/YaST2/y2log-1.gz \
  && echo "AUTOYAST OK"
```

Or even:

```shell
zgrep "Found users in profile with equal <uid>." /var/log/YaST2/y2log-1.gz
echo "AUTOYAST OK"
```